File: _release/PlainScripts/PositionScript.vb
Imports System
Imports System.Windows.Forms
Imports GTA

Public Class PositionScript
   Inherits Script

   Private Sub PositionScript_KeyDown(ByVal sender As Object, ByVal e As GTA.KeyEventArgs) Handles MyBase.KeyDown
      If e.Key = Keys.Home Then Game.DisplayText(Player.Character.Position.ToString)
   End Sub

End Class

File: _release/PlainScripts/PositionScript.cs
using System;
using System.Windows.Forms;
using GTA;

public class PositionScript : Script {

   public PositionScript() {
      this.KeyDown += new GTA.KeyEventHandler(this.PositionScript_KeyDown);
   }

   private void PositionScript_KeyDown(object sender, GTA.KeyEventArgs e) {
      if (e.Key == Keys.Home) Game.DisplayText(Player.Character.Position.ToString());
   }

}